src/content/docs/faqs/common-questions-for-founders.md

```diff
@@ -1,6 +1,66 @@
 ---
-title:  "Common Questions for Founders"
-description:  "Common Questions for Founders"
+title: "Founders FAQ"
+description: "Frequently Asked Questions tailored for founders using Angor."
 sidebar:
-  order: 2
+  order: 3
 ---
+
+## **What is Angor, and how can it help me as a founder?**
+Angor is a decentralized crowdfunding protocol that offers secure funding channels for projects. It helps founders attract serious backers by guaranteeing fund security, which fosters trust among investors.
+
+![bitcoin](./bitcoin2.svg)
+
+## **How do I create a project on Angor?**
+To create a project:
+1. Register the project through Angor App.
+2. Define milestones and funding goals.
+3. Use Angor Hub to update investors and engage with the community.
+
+![bitcoin](./bitcoin2.svg)
+
+## **How does Angor ensure fund security for investors?**
+Angor utilizes Bitcoin smart contracts to:
+- Lock funds based on milestones.
```

Comment:
also mention maybe that only the investor and founder keys can access the funds depending on the milestone, no middle man or third party is involved.